MozillaFoundation/foundation.mozilla.org
# Wagtail upgraded to 6.3.1

# Description

This PR introduces an upgrade of Wagtail `5.2.6` to `6.3.1`. This upgrade brings with it some version dependencies of other packages:

- `django-taggit==6.1.0`, previously `3.0.0`
- `wagtail-inventory==3.1`, previously `2.5`
- `willow[heif]==1.8.0`, previously `1.6.3`

Link to sample test page: 
Related PRs/issues: https://mozilla-hub.atlassian.net/browse/TP1-1318

**Tests**
- Pull this branch
- Build your docker container again to apply the new dependencies versions.
- Make sure the conatiner built with no errors and the application is running correctly.
- Access the CMS and confirm you can navigate correctly.
- Open the site and confirm everything was rendered properly and with no errors.
